sdl2promela/sdl/model.py

```diff
@@ -907,17 +917,23 @@ def convert(source: ogAST.PrimSequenceOf):
     for elem in source.value:
         elements.append(convert(elem))
 
-    return SequenceOf(elements, source.exprType)
+    sequenceof = SequenceOf(elements, source.exprType)
```

Comment:
Type assignment cannot be handled in the constructor?

Reply:
It was easier to modify (base class and places where the objects are created)